background.js

```diff
@@ -16,6 +16,43 @@ function tabCallback(tabId, changeInfo, updatedTab) {
   }
 }
 
+function handleMaybeInt(maybeString) {
+  const maybeInt = parseInt(maybeString, 10);
+  if (Number.isNaN(maybeInt)) {
+    return null;
+  }
+  return maybeInt;
+}
+
+function handlePassphraseCleaning() {
+  chrome.storage.local.get(["lastModified", "lockAfterSeconds", "temporaryPassphrase"], function(data) {
+    const currentTime = Date.now();
+    const lastModified = handleMaybeInt(data.lastModified);
+    const lockAfterSeconds = handleMaybeInt(data.lockAfterSeconds);
```

Comment:
The reason is that localStorage only stores strings, while chrome.storage.local stores any JSON serializable string.

We could store a `JSON.stringify` string in localStorage and parse it on retrieve, however Elm ports are also only handling Strings, so we dedided to only store string on the backend, but then we used flags, which allow number.

I have no strong opinion about moving to localStorage + json.Stringify if you think it might worth it.